test/detect/php/default.txt

```diff
@@ -48,6 +48,16 @@ enum Foo: string {
     case Test = 'test';
 }
 
+function read(mixed $key): string {
```

Comment:
IE: See the failing tests "AssertionError: default.txt should be detected as php, but was zephir".

This is likely because you added code here that "looks like" Zephir (which is very close to PHP).

I'd suggest reverting this part of your changes and seeing if that makes tests green.

Reply:
I will try this, thank you. I used that particular snippet because it has the new `mixed` type in it, but I think at least for the `default.txt` file, the simpler, the better.

Reply:
`default.txt` should in practice be a "clear, representative sample" of a language... no more, no less.  So perhaps they should change indeed slowly over time, but the current system makes it frustrating to do so.

The current auto-detect system is all relative to other languages vs being more yes/no... if it was a more isolated and binary decision such as "this is PHP" or "this is not" these tests would make a lot more sense.

Our sample size is really too small for what we're attempting to do with it anyways - so it's become a very artificial thing.